CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Introduction
+
+Let's start with the relationship between the two important components:
+
+* **Kubernetes** is a container orchestration system, and is used to run
+  containers on a cluster
+* **KubeVirt** is an add-on which is installed on-top of Kubernetes, to be able
+  to add basic virtualization functionality to Kubernetes.
+
+Even though KubeVirt is an add-on to Kubernetes, both of them have things in
+common:
+
+* Mostly written in golang
+* Often related to distributed microservice architectures
+* Declarative and Reactive (Operator pattern) approach
+
+This short page shall help to get started with the projects and topics
+surrounding them
+
+
+## Contributing to KubeVirt
+
+### Our workflow
+
+Contributing to KubeVirt should be as simple as possible. Have a question? Want
+to discuss something? Want to contribute something? Just open an
+[Issue](https://github.com/kubevirt/kubevirt/issues), a [Pull
+Request](https://github.com/kubevirt/kubevirt/pulls), or send a mail to our
+[Google Group](https://groups.google.com/forum/#!forum/kubevirt-dev).
+
+If you spot a bug or want to change something pretty simple, just go
+ahead and open an Issue and/or a Pull Request, including your changes
+at [kubevirt/kubevirt](https://github.com/kubevirt/kubevirt).
+
+For bigger changes, please create a tracker Issue, describing what you want to
+do. Then either as the first commit in a Pull Request, or as an independent
+Pull Request, provide an **informal** design proposal of your intended changes.
+The location for such propoals is
+[/docs](https://github.com/kubevirt/kubevirt/tree/master/docs) in the KubeVirt
```

Comment:
we might want to consider a sub directory for proposals in our top level docs directory.